lit_tests/kernel/wave/codegen.py

```diff
@@ -2231,7 +2231,7 @@ def scalar_codegen_i32(
 
     # Final dispatch args dtype
     # CHECK: flow.dispatch @scalar_codegen_i32::@scalar_codegen_i32(
-    # CHECK-SAME: %arg0, %arg1, %arg2, %arg3)
+    # CHECK-SAME: %{{.*}}, %{{.*}}, %arg2, %arg3)
```

Comment:
Same: Why is this necessary for arg0 and arg1?

Reply:
Does it then make sense to match these using SSA names as well? Or possibly checking that the DAG is correct, i.e. matching that the SSA value resulting from a specific op is used here?